components/api/services/containers/runtime/v1/runtime.proto

```diff
@@ -0,0 +1,43 @@
+syntax = "proto3";
+
+package moby.core.services.containers.runtime.v1;
+
+option go_package = "runtime";
+
+import "github.com/moby/moby/components/api/types/container/container.proto";
+import "google/protobuf/duration.proto";
+
+service ContainerRuntimeService {
```

Comment:
Similarly, can we just expose (almost) bare containerd service? (with some `labels` and `extensions`)
https://github.com/containerd/containerd/blob/master/api/services/containers/v1/containers.proto